pkg/container/bubblewrap_runner.go

```diff
@@ -167,6 +189,10 @@ func (bw *bubblewrap) StartPod(ctx context.Context, cfg *Config) error {
 	ctx, span := otel.Tracer("melange").Start(ctx, "bubblewrap.StartPod")
 	defer span.End()
 
+	if err := bw.assertUnshareUser(ctx); err != nil {
```

Comment:
Would it make sense to run this in `TestUsability` instead of StartPod?

Reply:
yeah, that makes more sense.  I didn't know about testusability.
changed to that.